sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/models/SettingSelector.java

```diff
@@ -47,15 +47,8 @@ public SettingSelector() {
     /**
      * Gets the expressions to filter {@link ConfigurationSetting#getKey() keys} on for the request.
      *
-     * <p>
-     * Examples:
-     * <ul>
-     * <li>If {@code keyFilter = "*"}, settings with any key are returned.</li>
-     * <li>If {@code keyFilter = "abc1234"}, settings with a key equal to "abc1234" are returned.</li>
-     * <li>If {@code keyFilter = "abc*"}, settings with a key starting with "abc" are returned.</li>
-     * <li>If {@code keyFilter = "*abc*"}, settings with a key containing "abc" are returned.</li>
-     * <li>If {@code keyFilter = "abc,def"}, settings with a key equal to "abc" or "def" are returned.</li>
-     * </ul>
+     * <p>See <a href="https://github.com/Azure/AppConfiguration/blob/master/docs/REST/kv.md#filtering">Filtering</a>
```

Comment:
Does AppConfiguration have Microsoft Docs? I'd rather link to that than their GitHub page since those will be more official.

Reply:
their doc hasn't been updated yet. https://docs.microsoft.com/en-us/azure/azure-app-configuration/concept-key-value
The service team will update it when GA.

Reply:
In .net we also use github page.